### ISSUE TITLE
Adding animation on scroll on every section as the original website have and change the footer drop email section

### ISSUE THREAD

**Issue by 123shreyanvi** [ASHISH26940/toogoodto#47]
adding animation on scroll will make more attractive

**Comment by ASHISH26940** [comment]
Yeah sure why not. It's a good Idea.

But do comment here your the animation library you are going to use. For convenience.

Recommended:
Vanilla CSS
GSAP

it's up to you but do mention any animation library which will be used.

**Comment by 123shreyanvi** [comment]
I'll mostly use plain css

**Comment by ASHISH26940** [comment]
Yeah you can start working on it

**Comment by ASHISH26940** [comment]
whats the progress and discord id

**Comment by lord-cyclone100** [comment]
@ASHISH26940 i will add aos effects. pls assign me

**Comment by rko0211** [comment]
Kindly assign me this one, i can do it in time

**Comment by ASHISH26940** [comment]
Yeah you can start working on it and your recent Pr I will be accepting it later
